etc/cn-full-test.toml

```diff
@@ -0,0 +1,92 @@
+# service node type, [DN|CN|LOG]
+service-type = "CN"
+
+[log]
+level = "debug"
+format = "json"
+max-size = 512
+
+[cn.frontend]
+#root name
+rootname = "root"
+
+#dump user name
+dumpuser = "dump"
+
+#dump user password
+dumppassword = "111"
+
+#port defines which port the mo-server listens on and clients connect to
+port = 6001
+
+#listening ip
+host = "0.0.0.0"
+
+#host mmu limitation. 1 << 40 = 1099511627776
+hostMmuLimitation = 1099511627776
+
+#guest mmu limitation.  1 << 40 = 1099511627776
+guestMmuLimitation = 1099511627776
+
+#mempool maxsize.  1 << 40 = 1099511627776
+mempoolMaxSize = 1099511627776
+
+#mempool factor.
+mempoolFactor = 8
+
+#process.Limitation.Size.  10 << 32 = 42949672960
+processLimitationSize = 42949672960
```

Comment:
It is not really necessary to expose this information, you can get the machine's resource information at startup and calculate a reasonable value

Reply:
I remove it